lib/content-services/src/lib/search/components/search-date-range/search-date-range.component.ts

```diff
@@ -107,16 +107,16 @@ export class SearchDateRangeComponent implements SearchWidget, OnInit, OnDestroy
 
         if (this.settings?.maxDate) {
             if (this.settings.maxDate === 'today') {
-                this.maxDate = this.dateAdapter.today().endOf('day');
+                this.maxDate = endOfToday();
```

Comment:
should be using adapter as before

```
this.maxDate = endOfDay(this.dateAdapter.today())
```

Reply:
![endOfDay](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/683e5ce0-195a-41a4-8dd6-c0fd22d7ffb7)